### PR TITLE
[DOXIASITETOOLS-267] Upgrade to Doxia 2.0.0-M4

### PR DESCRIPTION
**Pull request by michael-o**
This closes #64

### CODE REVIEW

File: doxia-site-renderer/src/test/resources/site/fml/faq.fml [hunk 1]
```diff
@@ -20,7 +20,7 @@ under the License.
 
 <faqs xmlns="http://maven.apache.org/FML/1.0.1"
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/FML/1.0.1 file:../../../../../../../doxia/doxia-modules/doxia-module-fml/src/main/resources/fml-1.0.1.xsd"
```
Comment (hboutemy) [inline doxia-site-renderer/src/test/resources/site/fml/faq.fml:23]: I'm not convinced removing xdoc-2.0.xsd and fml-1.0.1.xsd copies are a good idea: it adds a network dependency during build

at least this should be a separate issue: this is not about upgrading Doxia 2.0 M3 to M4

Comment (michael-o) [reply doxia-site-renderer/src/test/resources/site/fml/faq.fml:23]: No, it is not. Doxia bundles them now and the resource resolver will pick then up from classpath. See: https://github.com/apache/maven-doxia/pull/115/files#diff-262a682f07b45aeb2cacb62700410586a3e522010b21edfa3f7d213e62d2794bR701-R714

Comment (michael-o) [reply doxia-site-renderer/src/test/resources/site/fml/faq.fml:23]: https://issues.apache.org/jira/browse/DOXIA-669

Comment (michael-o) [reply doxia-site-renderer/src/test/resources/site/fml/faq.fml:23]: grafik

Comment (hboutemy) [reply doxia-site-renderer/src/test/resources/site/fml/faq.fml:23]: oh, great, then ok, wrong evaluation from me: this is a really nice improvement from Doxia 2.0.0-M4

**Review by hboutemy — APPROVED** [review]
LGTM

File: doxia-site-renderer/src/test/resources/site/fml/faq.fml (hunk 1 above)
Comment (michael-o) [reply doxia-site-renderer/src/test/resources/site/fml/faq.fml:23]: This also expects that Doxia Core is physically present to Sitetools which is not always the case.

**Pull request closed without merging**